stimela/backends/flavours/python_flavours.py

```diff
@@ -119,23 +119,33 @@ def get_arguments(self, cab: Cab, params: Dict[str, Any], subst: Dict[str, Any],
                         ).decode('ascii')
 
         # form up command string
+        if stimela.VERBOSE:
+            msg1 = f"""print("## importing {py_module}.{py_function}")"""
```

Comment:
```suggestion
            msg1 = f"""print(f"## importing {py_module}.{py_function}")"""
```

Missing f-string?

Reply:
No, ``py_module`` and ``py_function`` are variables in the outer scope, so the outer f-string takes care of substituting them.

It's a bit confusing with f-strings used to form up code that also uses f-strings inside, but I think this is correct.

Reply:
Thank you. It should be ready to merge.